<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_FirmWare</class>
 <widget class="QDialog" name="Dialog_FirmWare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="PushButton_FirmWare_Updata">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: qconicalgradient(cx:0.5, cy:0.522909, angle:179.9,
		stop:0.494318 rgba(214, 214, 214, 255), stop:0.5 rgba(236, 236, 236, 255));
	border: 1px solid rgb(124, 124, 124);
	border-radius:2px;
}

QPushButton:pressed{
	background-color: qconicalgradient(cx:0.5, cy:0.522909, angle:179.9,
		stop:0.494318 rgba(134, 198, 233, 255), stop:0.5 rgba(206, 234, 248, 255));
	border: 1px solid #5F92B2;
}

QPushButton:hover{
background-color: qconicalgradient(cx:0.5, cy:0.522909, angle:179.9,
	stop:0.494318 rgba(181, 225, 250, 255), stop:0.5 rgba(222, 242, 251, 255));
border: 1px solid #3C80B1;
}</string>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_FirmWare_Msg">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;提示消息:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="ProgressBar_FirmWare_ProgressBar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>280</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton_FirmWare_Complete">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>410</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: qconicalgradient(cx:0.5, cy:0.522909, angle:179.9,
		stop:0.494318 rgba(214, 214, 214, 255), stop:0.5 rgba(236, 236, 236, 255));
	border: 1px solid rgb(124, 124, 124);
	border-radius:2px;
}

QPushButton:pressed{
	background-color: qconicalgradient(cx:0.5, cy:0.522909, angle:179.9,
		stop:0.494318 rgba(134, 198, 233, 255), stop:0.5 rgba(206, 234, 248, 255));
	border: 1px solid #5F92B2;
}

QPushButton:hover{
background-color: qconicalgradient(cx:0.5, cy:0.522909, angle:179.9,
	stop:0.494318 rgba(181, 225, 250, 255), stop:0.5 rgba(222, 242, 251, 255));
border: 1px solid #3C80B1;
}</string>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton_FirmWare_SeleteFile">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: qconicalgradient(cx:0.5, cy:0.522909, angle:179.9,
		stop:0.494318 rgba(214, 214, 214, 255), stop:0.5 rgba(236, 236, 236, 255));
	border: 1px solid rgb(124, 124, 124);
	border-radius:2px;
}

QPushButton:pressed{
	background-color: qconicalgradient(cx:0.5, cy:0.522909, angle:179.9,
		stop:0.494318 rgba(134, 198, 233, 255), stop:0.5 rgba(206, 234, 248, 255));
	border: 1px solid #5F92B2;
}

QPushButton:hover{
background-color: qconicalgradient(cx:0.5, cy:0.522909, angle:179.9,
	stop:0.494318 rgba(181, 225, 250, 255), stop:0.5 rgba(222, 242, 251, 255));
border: 1px solid #3C80B1;
}</string>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>511</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;固件更新中请不要断电，不要关闭此窗口！！！&lt;/p&gt;&lt;p&gt;1：点击选择文件从本地电脑中找到要更新的.bin文件。&lt;/p&gt;&lt;p&gt;2：选好.bin文件后点击更新，等待2分钟，直至出现提示“固件更新完成”。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PushButton_FirmWare_Complete</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_FirmWare</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
